src/components/modals/login-modal/ForgotPasswordContent.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button, Form, Input } from 'antd';
+import React from 'react';
+import './forgot-password-content.less';
+
+interface Props {
+  switchToConfirmationPage: () => void;
+}
+
+const ForgotPasswordContent: (props: Props) => JSX.Element = (props: Props) => {
+  return (
+    <p>
```

Comment:
We shouldn't next inside <p> tags, and we ought to use AntD typography instead of <p> tags anyways.